### DISCUSSION TITLE
URL for SlideOvers Forms

### DISCUSSION THREAD

**Discussion by SayHeyD** [filamentphp/filament#13995] [Ideas]
I would like it to be possible to have a simple Resource, were the edit, view and create forms in the modals have an according url. 

Essentially when accessing the edit url `/{record}/edit` i want to land on the index page but have the edit modal open and the edit URL shown in the address bar.

Something similar is possible with [Tabs](https://filamentphp.com/docs/3.x/infolists/layout/tabs#persisting-the-current-tab-in-the-urls-query-string). A solution with a query string would also be totally fine.

**Comment by howdu** [comment]
https://filamentphp.com/docs/3.x/panels/resources/getting-started#generating-urls-to-resource-modals

**Reply by SayHeyD** [reply]
Oh my god, you don't know how long i have been looking for this but it never came across my mind to look into the panel documentation. Thank you <3
